yarn.lock

```diff
@@ -30362,6 +30350,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"printj@npm:~1.1.0":
+  version: 1.1.2
+  resolution: "printj@npm:1.1.2"
+  bin:
+    printj: ./bin/printj.njs
+  checksum: 10/45376a5ee7ef2e0d7ff0b4fecc893d73995a332e63d7e0622a544fe662c8213d22f0c9750e627c6d732a7d7a543266be960e6cd51cf19485cce87cf80468bb41
+  languageName: node
+  linkType: hard
```

Comment:
Where are we using this? In the btc or sol snap?

Reply:
Seems to be pulled of by `crc-32`, which got downgraded here (in this PR) for some reason...

Looks like there's a resolution that affects this. Not sure how to track that now, let me check.